### ISSUE TITLE
Inquiry Regarding Refined DoTA Labels and DAPT Ablation

### ISSUE THREAD

**Issue by Smartpearkorl** [tue-mps/simple-tad#2]
Congratulations on your paper “Simplifying Traffic Anomaly Detection with Video Foundation Models” accepted at ICCV Workshops. I am writing to kindly ask for clarification on two points mentioned in the paper:

In the paper, you stated: “Although only about 1% of the clips were refined, we release the corrected annotations for reproducibility. We use the refined DoTA dataset for comparisons between Video ViT models, and the original dataset when comparing to existing specialized methods.” However, I could not find the refined DoTA annotations in the project repository. Could you please explain what criteria or rules were used to refine the DoTA dataset labels and how we can obtain the refined annotations?

Regarding the DAPT ablation, the paper presents results under a training-free setting. Have you also compared the performance of models with and without DAPT under the fully supervised setting on the DoTA and D2K datasets?

**Comment by orsveri** [comment]
Hello! 

Regarding refined annotations:
We first identified potentially problematic clips (more details in the supplementary). Then we manually went through them, checking for obvious deviation from the rules, explained in the original DoTA paper: "we asked the annotators to estimate the anomaly start as the time when the anomaly was inevitable. The anomaly end was meant to be the time when all anomalous objects are out of the field of view or are stationary." We found a number of clips where the anomaly starts or ends were clearly off. For example, the start could be labeled before even the anomaly object (or anything that would mean an anomaly) was visible, or the end of the anomaly could be labeled even when there were people still lying on the road after the collision. We also found several cases of two different video clips in one file. 
This refinement is still subjective. We simply wanted to check if a small number of significant inconsistencies could lead to reduced performance, so removing those inconsistencies would boost the performance. We did get a slight improvement from that, but not very consistent across all our evaluation cases. 

I will share the instructions to prepare the dataset with the refined labels (and the refined labels themselves and split videos) in the coming weeks. 

We always fully fined-tuned our models on DoTA and D2K, and in our DAPT ablations, we only changed DAPT datasets, and performed DAPT followed by full fine-tuning with the identical configuration. I hope that answers the question!

**Comment by orsveri** [comment]
@Smartpearkorl, I noticed you are the author of PromptTAD (although unfortunately, I cannot identify which one out of 3 to contact you directly). Congratulations on publication in RA-L! Great work and thank you for your contribution! 
If you would like to discuss more details of the work or, maybe, possibilities to integrate our approaches, please write me: s.orlova@tue.nl

**Comment by Smartpearkorl** [comment]
Thank you very much for the detailed clarification.
